pootle/apps/pootle_language/models.py

```diff
@@ -173,14 +173,12 @@ def get_stats_for_user(self, user):
 
         return self.get_stats()
 
-    def get_children_for_user(self, user):
-        translation_projects = self.translationproject_set \
-                                   .for_user(user) \
-                                   .order_by('project__fullname')
-        user_tps = filter(lambda x: x.is_accessible_by(user),
-                          translation_projects)
-
-        return user_tps
+    def get_children_for_user(self, user, select_related=None):
+        tps = self.translationproject_set.for_user(
+            user, select_related=select_related)
+        return filter(
```

Comment:
well i generally go with the list comprehension over functional style, but...

there are times when any, all, filter etc are imho more expressive - mostly where the condition is significant, and there is no transformation to the list members.